packages/tx-construction/src/tx-builder/types.ts

```diff
@@ -284,12 +293,15 @@ export interface TxBuilder {
 export interface TxBuilderDependencies {
   inputSelector?: InputSelector;
   inputResolver: Cardano.InputResolver;
-  bip32Account: Bip32Account;
+  bip32Account?: Bip32Account;
   witnesser: Witnesser;
   txBuilderProviders: TxBuilderProviders;
   logger: Logger;
   outputValidator?: OutputBuilderValidator;
   handleProvider?: HandleProvider;
 }
 
-export type FinalizeTxDependencies = Pick<TxBuilderDependencies, 'bip32Account' | 'witnesser'>;
+export type FinalizeTxDependencies = { dRepPublicKey?: Crypto.Ed25519PublicKeyHex } & Pick<
```

Comment:
The commit message is a bit misleading. The breaking change is that `FinalizeTxDependencies` no longer requires a `bip32Account`, but should provide a dRepPublicKey if available.
`TxBuilder` `bip32Account` as optional dependency is not a breaking change.

Reply:
Commit message adjusted